src/com/service/PersonServiceTest.java

```diff
@@ -10,42 +10,79 @@
 
 class PersonServiceTest {
 
-    private static Person person;
-    private static PersonService personService;
-    private static String name;
-    private static EGender gender;
-    private static Date dateOfBirth;
-    private String interest;
-    private static EStatus status;
+    private Person person;
+    private PersonService classUnderTest;
 
     @org.junit.jupiter.api.BeforeEach
     void setup() {
-        person = new Person(name, gender, dateOfBirth, interest, status);
+        person = new Person("name", EGender.MALE, new Date(), "sdfdsfdsfds", EStatus.IN_RELATIONSHIP);
+        classUnderTest = new PersonService();
     }
 
     @org.junit.jupiter.api.Test
     void addNewPerson() {
-        fail("Yet to be implemented");
+        assertTrue(classUnderTest.addNewPerson(person));
+    }
+
+    @org.junit.jupiter.api.Test
+    public void givenPersonWithRealData_whenCreatingPerson_thenExpectToBeAddedInList() {
+        Person person = new Person("", EGender.MALE, new Date(), "dsfsd", EStatus.IN_RELATIONSHIP);
+    }
+
+    @org.junit.jupiter.api.Test
+    public void givenNullablePerson_whenCreatingPerson_thenExpectNotToBeAddedInList() {
+        assertFalse(classUnderTest.addNewPerson(null));
     }
 
     @org.junit.jupiter.api.Test
     void removePerson() {
```

Comment:
given_when_then.. apply it to the other methods and don't forget to stay consistent with your code